### PR TITLE
Fix F5 debug configuration JSON parse error.

### PR DESCRIPTION
**Pull request by sean-mcmanus**
Fix F5 debugging regression from https://github.com/microsoft/vscode-cpptools/pull/8368 . json.Parse was throwing an exception (SyntaxError: Unexpected token in JSON at position 538).

### CODE REVIEW

File: Extension/src/Debugger/configurationProvider.ts [hunk 1]
```diff
@@ -563,7 +563,7 @@ class WindowsConfigurationProvider extends DefaultConfigurationProvider {
         "ignoreFailures": true
     },
     {
-        "description":  "${localize("enable.intel.disassembly.flavor", "Set Disassembly Flavor to {0}", "Intel").replace(/\"/g, "\\\"")},
+        "description":  "${localize("enable.intel.disassembly.flavor", "Set Disassembly Flavor to {0}", "Intel").replace(/\"/g, "\\\"")}",
```
Comment (bobbrow) [inline Extension/src/Debugger/configurationProvider.ts:566]: There's a lot of double quotes in there.  Can we change the strings to use single quotes in this case?

File: Extension/src/Debugger/configurationProvider.ts [hunk 2]
```diff
@@ -602,7 +602,7 @@ class LinuxConfigurationProvider extends DefaultConfigurationProvider {
...
```
Comment (bobbrow) [inline Extension/src/Debugger/configurationProvider.ts:605]: Same here. Can we use single quotes?

**Review by bobbrow — APPROVED** [review]

File: Extension/src/Debugger/configurationProvider.ts (hunk 1 above)
Comment (bobbrow) [reply Extension/src/Debugger/configurationProvider.ts:566]: And, I'm just referring to the outermost quotes to change to single quotes

Comment (sean-mcmanus) [reply Extension/src/Debugger/configurationProvider.ts:566]: Why? I don't recall seeing JSON with single quotes....that might only work with TypeScript/JavaScript strings?

Comment (sean-mcmanus) [reply Extension/src/Debugger/configurationProvider.ts:566]: The whole string is part of a `` interpolated string.

Comment (WardenGnaw) [reply Extension/src/Debugger/configurationProvider.ts:566]: Its similar to the `description` field above. 

One way to fix this string mess is to move it to be a javascript object and ToString it to the caller.

**Review by WardenGnaw — APPROVED** [review]

File: Extension/src/Debugger/configurationProvider.ts (hunk 1 above)
Comment (bobbrow) [reply Extension/src/Debugger/configurationProvider.ts:566]: My bad. I thought this was a JavaScript object.  No need to make changes then. I already approved.

**Comment by sean-mcmanus** [comment]
Well, I'll check this in, but feel free to refactor the string stuff later if you want.

**Pull request merged**